src/composeSpecs.ts

```diff
@@ -114,10 +114,16 @@ function mergePs315(chain: Ps315Chain): TPs315Options | 'Off' {
     acc = { ...acc, ...cur }
 
     if (cur.cleanDescriptorsExceptions !== undefined) {
-      acc.cleanDescriptorsExceptions = concatUnique(
-        prevCleanDesc,
-        cur.cleanDescriptorsExceptions,
-      )
+      // If current spec explicitly sets an empty array, respect that (don't concatenate)
+      // Otherwise, concatenate with previous values
+      if (cur.cleanDescriptorsExceptions.length === 0) {
+        acc.cleanDescriptorsExceptions = []
+      } else {
+        acc.cleanDescriptorsExceptions = concatUnique(
+          prevCleanDesc,
+          cur.cleanDescriptorsExceptions,
+        )
+      }
```

Comment:
On 169466aa04e, resetting cleanDescriptorsExceptions on empty array, what‘s the backward compatibility aspect? If the variable is `[]` in all constituents to a composite spec, it will remain `[]` in the end.

While I see an advantage of this as a `reset` mechanism, my concern is that that specific behavior differs from the other enrichments such as `retainPatientCharacteristics` or `hostProps`, possibly making it unintuitive.

Alternatively we could introduce explicit `false | []` as the type like with `retainPatientCharacteristics` once/if this is needed and leave `[]` as 'not extending previous definition'?